common/secrets.js

```diff
@@ -96,7 +96,7 @@ const POSTCODES_API_KEY =
 const S3_KMS_KEY_ID =
     process.env.S3_KMS_KEY_ID || getParameter('s3.kms.key.id');
 
-const DATA_STUDIO_AFA_URL = getParameter('dataStudio.url.awardsForAll');
+const DATA_STUDIO_UNDER10K_URL = getParameter('dataStudio.url.awardsForAll');
```

Comment:
We could probably stand to change `dataStudio.url.awardsForAll` at some-point, but this is fine for now.